packages/main/src/models/conversation.ts

```diff
@@ -35,28 +35,33 @@ export function getReplyParticipants(
 ): Participants {
   const senderAddress = Addr.build(sender)!
   const participants = getParticipants(conversation)
-  const to = uniqBy(
-    Addr.normalizedEmail,
-    List(
-      participants
-        .replyTo!.concat(participants.to)
-        .filter(p => !Addr.equals(senderAddress, p))
-    )
-  )
-    .sortBy(Addr.formatAddress)
-    .toArray()
+  const to = participants.replyTo
+    ? uniqBy(
+        Addr.normalizedEmail,
+        List(
+          participants.replyTo
+            .concat(participants.to)
+            .filter(p => !Addr.equals(senderAddress, p))
+        )
+      )
+        .sortBy(Addr.formatAddress)
+        .toArray()
+    : []
 
-  const cc = uniqBy(
-    Addr.normalizedEmail,
-    List(
-      participants.cc.filter(
-        p =>
-          !Addr.equals(senderAddress, p) && !to.some(p_ => Addr.equals(p, p_))
+  const cc = to
```

Comment:
You know that `to` will be defined, even if it might be an empty array, so this ternary is not necessary. Remember that `[]` is truthy, so the false branch of this conditional will never run.